Packs/MitreCaldera/Integrations/MitreCaldera/MitreCaldera.py

```diff
@@ -0,0 +1,1885 @@
+    sleep_max = args.get('sleep_max')
+    pending_contact = args.get('pending_contact')
+    group = args.get('group')
+
+    response = client.update_agent(paw, watchdog, sleep_min,
+                                            trusted, sleep_max, pending_contact, group)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Agents',
+        outputs_key_field='',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def update_fact_source_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    fact_source_id = args.get('fact_source_id')
+    source_name = args.get('source_name')
+    source_adjustments = argToList(args.get('source_adjustments', []))
+    source_relationships = argToList(args.get('source_relationships', []))
+    source_id = args.get('source_id')
+    source_rules = argToList(args.get('source_rules', []))
+    source_facts = argToList(args.get('source_facts', []))
+    source_plugin = args.get('source_plugin')
+
+    response = client.update_fact_source(fact_source_id, source_name, source_adjustments,
+                                                         source_relationships, source_id, source_rules, source_facts, source_plugin)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Sources',
+        outputs_key_field='',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def update_objective_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    objective_id = args.get('objective_id')
+    name = args.get('name')
+    goals = argToList(args.get('goals', []))
+    description = args.get('description')
+
+    response = client.update_objective(
+        objective_id, name, goals, description)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Objectives',
+        outputs_key_field='',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def update_operation_fields_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    operation_id = args.get('operation_id')
+    obfuscator = args.get('obfuscator')
+    autonomous = args.get('autonomous')
+    state = args.get('state')
+
+    response = client.update_operation_fileds(
+        operation_id, obfuscator, autonomous, state)
+    command_results = CommandResults(
+        outputs_prefix='MitreCaldera.Operations',
+        outputs_key_field='id',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def update_main_config_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    property = args.get('property')
+    value = args.get('value')
+
+    client.update_main_config(property, value)
+    command_results = CommandResults(
+        readable_output=f"{property} updated to {value} in main config."
+    )
+    return_results(command_results)
```

Comment:
```suggestion
    return command_results
```
as this will used in the `return_results` in the `main`